### PR TITLE
Temporarily increase cgroup memory limits for eve kubevirt

### PR DESCRIPTION
**Pull request by zedi-pramodh**
1) Increased cgroup memory limits by 10 times.
2) Will be re-configured to proper limits in future commits after resizing the system.

Tested:
======
All k3s components starts well in kubevirt image.
No-op in kvm image.

### CODE REVIEW

File: pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup [hunk 1]
```diff
@@ -66,7 +71,11 @@ if [ -z "${ctrd_cgroup_cpus_limit}" ] || [ "${ctrd_cgroup_cpus_limit}" = "0" ];
 fi
 
 CGROUPS="cpuset cpu cpuacct blkio memory devices freezer net_cls perf_event net_prio hugetlb pids systemd "
-EVESRVICES="sshd edgeview wwan wlan lisp guacd pillar vtpm watchdog xen-tools newlogd "
```
Comment (deitch) [inline pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup:69]: If we already are fixing this, can we please correct the type for `EVESRVICES` to `EVESERVICES`?

File: pkg/grub/rootfs.cfg [hunk 2]
```diff
@@ -342,14 +348,30 @@ function set_xen_boot {
...
+   if [ "$arch" = "x86_64" ]; then
+      set_global dom0_flavor_tweaks "$dom0_flavor_tweaks crashkernel=2G-64G:128M,64G-1T:256M,1T-:512M"
+   fi
+   # see the set_global eve_flavor kvm comment in set_kvm_boot
+   set_global eve_flavor kubevirt
+}
+
 function set_eve_flavor {
    # if eve_flavor is not set explicitly we assume it is
    # embedded in the end of the name of the image
```
Comment (deitch) [inline pkg/grub/rootfs.cfg:365]: Is this a rational thing to do? I don't mean your specific change here; I mean the whole "tease it out from the image name". Is there any reason we cannot remove this all and say, "if eve_flavor is not set, that is an error"?

File: pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup [hunk 3]
```diff
@@ -5,9 +5,14 @@ if test -f /proc/vmcore; then
     exit 0;
 fi
 
-default_cgroup_memory_limit=838860800 #800M
+virttype=`cat /run/eve-virt-type`
 default_cgroup_cpus_limit=1
 
+if [ $virttype = "kubevirt" ]; then
```
Comment (deitch) [inline pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup:11]: This is one of 2 (more?) `if` special statements for `kubevirt`. Would it be better to capture them all into one place, make it a `case` statement for all known HV?

e.g.

```bash
case $virttype in
kubevirt)
  # set lots of things
kvm|xen)
  # set other things
*)
  # error
esac
```

Would that make this any clearer?

File: pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup (hunk 1 above)
Comment (zedi-pramodh) [reply pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup:69]: I can fix that.

File: pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup (hunk 3 above)
Comment (zedi-pramodh) [reply pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup:11]: We could do that. But this is just a temporary fix and need to be cleaned up anyway.

Comment (deitch) [reply pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup:11]: Fair enough. Even so, nothing is as permanent as something temporary. 😬

**Comment by zedi-pramodh** [comment]
Agreed. In fact I have to amend this commit once #3231 is merged since some changes are no longer required.

**Comment by zedi-pramodh** [comment]
This can be merged too since https://github.com/lf-edge/eve/pull/3231 is merged

**Comment by zedi-pramodh** [comment]
Wow I screwed up something in rebase and commit update. There should not be 60 files in the commit !! I will close this PR and open a new one. Sorry for the mess up.

**Comment by zedi-pramodh** [comment]
Cloned to https://github.com/lf-edge/eve/pull/3260

**Pull request closed without merging**